### DISCUSSION TITLE
Voice Privacy - Frequency Hopping

### DISCUSSION THREAD

**Discussion by merc07** [egzumer/uv-k5-firmware-custom#332] [Ideas]
The Baofeng UV5RH has this nice little option and it works well, could it be added ?

**Comment by gartnl** [comment]
It is not really FH, just an offset to the diplayed frequency it seems. Bit of a puzzle what they wanted to achieve with it.
Even if it was actual FH than some mechanism to synchronise sets would be needed. And other HAM's would complain because you ar all over the place ..

**Comment by merc07** [comment]
I assumed it was within the narrow band of the frequency still so would not cause any issues as doing a frequency scan with the function on/off gives the same result.
You just get a silent receive when the other radio has the function off...
